packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -32,6 +34,52 @@ export interface TableProps {
     streamSpecification?: StreamViewType;
 }
 
+export interface DynamoDBAutoScalingProps {
```

Comment:
This is extremely verbose.

Can you think of a way to allow us (and anyone) to offer predefined autoscaling strategies? Perhaps a simple class hierarchy with a bunch of subclasses and static methods for discoverability:

```ts
this.table.addReadAutoScaling(TableAutoScaling.onlyScaleUp({ max: 100 }));
```

Try to think what would be the most common case(s) and optimize for those to be as concise as possible, allowing people to customize as needed.

Reply:
created issue https://github.com/awslabs/aws-cdk/issues/644